pkg/mods/epm/epm.elv

```diff
@@ -1,3 +1,8 @@
+# Uncomment this line to make it easy to identify dependencies on external
+# commands:
+#
+# pragma unknown-command = disallow
```

Comment:
It's better to leave this pragma uncommented, and declare all external commands explicitly:

```elvish
pragma unknown-command = disallow
var git~ = $e:git~
var rsync = $e:rsync~
```

Reply:
I can live with that but I chose my approach to make it easy to identify any external commands used by the `epm` module using the documented pragma approach. Your approach makes it a tiny bit harder until something like my proposal in issue #1681 is implemented. Yes, I can just `grep e: **.elv` but that is prone to false positives. However, this has made me wonder if additional info should be provided to my proposed external command callback to make it easier to know the context in which the external command is being run; e.g., which module, script, or interactive context the external command is invoked.